nixos/modules/services/misc/exhibitor.nix

```diff
@@ -15,9 +15,11 @@ let
     election-port=${toString cfg.zkElectionPort}
     cleanup-period-ms=${toString cfg.zkCleanupPeriod}
     servers-spec=${concatStringsSep "," cfg.zkServersSpec}
-    auto-manage-instances=${lib.boolToString cfg.autoManageInstances}
+    auto-manage-instances=${toString cfg.autoManageInstances}
     ${cfg.extraConf}
   '';
+  # NB: toString rather than lib.boolToString on cfg.autoManageInstances is intended.
+  # Exhibitor expects it to be a 0/1 integer rather than a truthy/falsey string
```

Comment:
Updated the comment and verified that toString does what I expect. Also rebased against latest master.